bluesky/tests/test_bec.py

```diff
@@ -67,3 +69,14 @@ def test_with_baseline(RE, hw):
     sd = SupplementalData(baseline=[hw.det])
     RE.preprocessors.append(sd)
     RE(scan([hw.ab_det], hw.motor, 1, 5, 5))
+
+
+def test_underhinted_plan(RE, hw):
+    bec = BestEffortCallback()
+    RE.subscribe(bec)
+
+    @bpp.run_decorator()
+    def broken_plan(dets):
+            yield from bps.trigger_and_read(dets)
```

Comment:
8 blank spaces here?

Reply:
Thanks. Bad copy/paste is the culprit. Fixed and force-pushed.